tests/tests_enabled.yml

```diff
@@ -20,3 +20,7 @@
     - name: Check file counts in rsyslog.d
       assert:
         that: rsyslog_d_file_count.matched < 3
+
+    - name: Check __rsyslog_version is set
+      assert:
+        that: __rsyslog_version != "0.0.0"
```

Comment:
what does this change have in common with the rest of the commit?

Reply:
> what does this change have in common with the rest of the commit?

Note: This change is a fix for "is there a test for the problem being fixed here? I am surprised that the problem was not caught before merging #87".  And the discussion is on-going in the thread https://github.com/linux-system-roles/logging/pull/91#discussion_r389896570.

Reply:
I know, but what does this new test have in common with removing length filters from `rsyslog_files_actions` etc ? I.e. why to have a commit
```
 Simplifying jinja2 conditional expression:

  old> variable is defined and variable != []
  new> variable | d([])

Adding a test case to check __rsyslog_version is set properly.
```
when the "Adding a test case to check __rsyslog_version is set properly." part has nothing to do with the "Simplifying jinja2 conditional expression:" part? Please split the commit in two. (You can keep both in this PR.)

Reply:
Understood your question.  This change was made before the agreement made in the meeting.  Instead of fixing the issues raised in the review process in the same pr, I will open a new pr for them.  Actually, that was my question https://github.com/linux-system-roles/logging/pull/91#issuecomment-597248067 :)

Reply:
But the issue of not having a test for the change in this PR is related to this PR, so please add it to this PR. How would you test that the change is correct otherwise?

Reply:
We need to reach an agreement first on what is actually a good test, though.